client/src/components/Landing/index.js

```diff
@@ -40,41 +40,41 @@ function Landing() {
 
   return (
     <>
-      <section className='hero'>
```

Comment:
You're adding double quotes to this code as well.

Reply:
Not sure what happen to this. I did a git pull origin develop a couple times. Edison and Henry are even watching when I did that again before the PR. My local code uses a double quote. Anyway, I will re-push and see what happen.

Reply:
Some of these changes are not even mine. It was probably added when I did a "git pull origin develop". In the JSX we are inconsistent about double quotes and single quote. I thought we agreed to leave the double quotes in the JSX. Prettier wont' even reformat the JSX part.
 Look through the different JSX and you will find double quotes and single quotes all over the place